.circleci/config.yml

```diff
@@ -226,9 +235,9 @@ workflows:
       - publish-docker-mininet:
           requires:
             - publish-docker-build
-      - publish-docker-p4c-fpm:
-          requires:
-            - publish-docker-build
+      - publish-docker-p4c-fpm
+          # requires:
+            # - publish-docker-build
```

Comment:
Since the build times are much lower now, we could think about running this job on all branches, but only push on master. Maybe change the `docker_push` part to include a check?

Reply:
It sounds like a good idea to me. You could else try to build a P4 program (e.g., main.p4) as part of the docker image check

Reply:
I tied to get a short command, but it always resulted in this long string with lots of Bazel internal path in it.
We already compile main.p4 as part of CI (https://github.com/stratum/stratum/blob/master/stratum/pipelines/main/BUILD#L33-L41) so I'd leave that out for now.